cmd/keycloakb/keycloak_bridge.go

```diff
@@ -831,6 +831,11 @@ func main() {
 		}
 	}
 
+	// Tools for endpoint middleware
+	var idRetriever = keycloakb.NewRealmIDRetriever(keycloakClient)
+	var configurationReaderDBModule = configuration.NewConfigurationReaderDBModule(configurationRoDBConn, logger)
+	var endpointAvailabilityChecker = middleware.NewEndpointAvailabilityChecker(configuration.CheckKeyPhysical, idRetriever, configurationReaderDBModule)
```

Comment:
maybe endpointPhysicalCheckAvailabilityChecker ?

Reply:
Why not... even if I'm not fan of long variable names